### ISSUE TITLE
World Plants/Ferns: Incorrect interpretation of ranks (genera Bolophyta & Picrophyta as phyla)

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/testing#95]
From: Michael Hassler
Sent: Friday, April 16, 2021 9:03

Thanks. If you go to 
https://www.catalogueoflife.org/data/search?facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&limit=50&offset=0&sortBy=taxonomic

**there are a few grave errors, like synonyms of plant genera occurring as „phylum“ (!).** For example look for „Bolophyta Nutt.“.
I hope those are not my fault.

**At least one, „Picrophyta F. Muell.“, comes from one of my datasets and occurs there as regular genus synonym. Obviously your system registers every name which ends in „-phyta“ as Phylum…**

This is a major bug.

Michael

For attention of @gdower & @mdoering

**Comment by yroskov** [comment]
I do not see a rank in verbatim data:

![image](https://user-images.githubusercontent.com/28574252/115038365-171ca100-9e95-11eb-8586-92be13b07afb.png)

**Comment by yroskov** [comment]
I have tried to use Search for names with portion "phyta" in the Workbench with setting Partial, but got zero names delivered:

![image](https://user-images.githubusercontent.com/28574252/115038757-724e9380-9e95-11eb-9c43-78a46bd2e2f0.png)

Attempt to use wild card as *phyta aor %phyta also failed to get names.

**Comment by mdoering** [comment]
Yes, that is expected behavior. Partial is a prefix search, @thomasstjerne we should change the CLB UI labels like we did for the [portal already](https://www.catalogueoflife.org/data/search?facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&limit=50&offset=0&sortBy=taxonomic).

**Comment by mdoering** [comment]
The bad rank might indeed be an interpretation based on the name suffix. 

The root of all is the missing rank for synonyms in the WP archive - we flagged this before and I thought it had been solved by using the rank of the accepted name also for synonyms unless a different rank is clearly mentioned (I recall cases of subgenera or tribes).

Maybe we should turn off deriving ranks from names and instead (or additionally) add a new task to check [unranked names of type scientific](https://data.catalogueoflife.org/dataset/1141/names?facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&facet=authorship&facet=authorshipYear&facet=extinct&facet=environment&facet=origin&limit=50&nameType=scientific&offset=0&rank=unranked&sortBy=taxonomic). These should not exist in most cases. Only for placeholders, OTUs or rare cases when its really unknown.

**Comment by mdoering** [comment]
At least these names should have an "interpreted rank" issue flag
